### PR TITLE
(ASC-238) Verify the quotas have been configured properly

### PR DESCRIPTION
**Pull request by phongdly**

### CODE REVIEW

File: molecule/default/tests/test_verify_configured_quotas.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import os
+import pytest
+import re
+import testinfra.utils.ansible_runner
+import pdb
```
Comment (johnduarte) [inline molecule/default/tests/test_verify_configured_quotas.py]: I don't see this used anywhere? What is this for?

File: molecule/default/tests/test_verify_configured_quotas.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    output = run_on_host.run(cmd)
+    result = re.search(r'(?<=\s)[a-zA-Z0-9]+(?=\s)', output.stdout)
+    return result.group(0)
+
+
+def get_quota(quota_name, quota_show_output):
+    quota_name_regex = re.escape(quota_name) + r'(\s+\|\s+[0-9]+\s)'
+    if quota_name in quota_show_output:
+        # Getting the line of quota, such as 'instances                   | 9'
```
Comment (johnduarte) [inline molecule/default/tests/test_verify_configured_quotas.py]: The extra white space in this comment seems unnecessary.

**Review by johnduarte — CHANGES_REQUESTED** [review]
2 nit picks, but this looks good. I will be nice to get these helper methods into a shared library at some point.

File: molecule/default/tests/test_verify_configured_quotas.py (hunk 1 above)
Comment (phongdly) [reply molecule/default/tests/test_verify_configured_quotas.py]: ah, I added it in for debugging and forgot to remove it. I'll update it in the next push

File: molecule/default/tests/test_verify_configured_quotas.py (hunk 2 above)
Comment (phongdly) [reply molecule/default/tests/test_verify_configured_quotas.py]: I'll update it in the next push

**Review by johnduarte — APPROVED** [review]
👍

**Pull request merged**